libknet/Makefile.am

```diff
@@ -17,7 +17,7 @@ EXTRA_DIST		= $(SYMFILE)
 
 SUBDIRS			= . tests man
 
-libversion		= 1:0:0
+libversion		= 2:0:1
```

Comment:
what´s the equivalent soname for this one? in theory adding an API should bump to so.1.1.0 IIRC. we are not break backward compatibility.

Reply:
Yes, and that's exactly what happens in this case.

Reply:
ok, i guess i´ll need to get used to "weird" numbers ;)

I am going to merge https://github.com/kronosnet/kronosnet/pull/136 and then we can rebase this branch on top of master and ack/merge.

Reply:
They are weird indeed. I've seen several projects quote (or at least link to) the libtool manual in a comment right above their version info... and to be honest I'm inclined to do the same.